iothub/device/src/IotHubConnection.cs

```diff
@@ -475,29 +467,15 @@ static AmqpSettings CreateAmqpSettings()
             return amqpSettings;
         }
 
-        protected static AmqpLinkSettings SetLinkSettingsCommonProperties(AmqpLinkSettings linkSettings, TimeSpan timeSpan, ProductInfo productInfo)
+        protected static AmqpLinkSettings SetLinkSettingsCommonProperties(AmqpLinkSettings linkSettings, TimeSpan timeSpan, ProductInfo productInfo, string corrId, string linkType)
         {
             linkSettings.AddProperty(IotHubAmqpProperty.TimeoutName, timeSpan.TotalMilliseconds);
-
             linkSettings.AddProperty(IotHubAmqpProperty.ClientVersion, productInfo.ToString());
+            linkSettings.AddProperty(IotHubAmqpProperty.ChannelCorrelationId, linkType + ":" + corrId);
```

Comment:
Instead of passing these two values, why not just take the correlationId (already built)?

Reply:
Previously, the linktype was being sent, too. See the old code in IotHubConnection.cs:
```cSharp
linkSettings.AddProperty(IotHubAmqpProperty.ChannelCorrelationId, "twin:" + corrId);
```

I'm not sure if the link type is necessary, but for now I'd rather not remove it altogether.